### PR TITLE
Summingbird changes for Algebird 0.13.0

### PR DESCRIPTION
**Pull request by piyushnarang**
Noticed that the Algebird bump broke SB in a couple of places. We might need to also bump Scalding as part of this (which I can do when we have a release out).

### CODE REVIEW

**Comment by piyushnarang** [comment]
@pankajroark / @johnynek / @ttim - please take a look when you can, thanks.

File: summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala [hunk 1]
```diff
@@ -81,7 +79,7 @@ class HDFSStateLaws extends WordSpec {
     }
   }
 
-  def leftClosedRightOpenInterval(low: Timestamp, high: Timestamp) = Interval.leftClosedRightOpen[Timestamp](low, high).right.get
+  def leftClosedRightOpenInterval(low: Timestamp, high: Timestamp) = Interval.leftClosedRightOpen[Timestamp](low, high)
```
Comment (johnynek) [inline summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala]: can we put a type on the result here? `Interval[Timestamp]`

**Review by johnynek — CHANGES_REQUESTED** [review]

File: summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala (hunk 1 above)
Comment (piyushnarang) [reply summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala]: sure

**Comment by johnynek** [comment]
👍

File: summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala [hunk 2]
```diff
@@ -16,15 +16,13 @@
 
 package com.twitter.summingbird.batch
 
-import java.util.{ TimeZone, UUID }
-
-import com.twitter.algebird.{ Intersection, Interval }
-import com.twitter.scalding.{ DateParser, RichDate }
+import java.util.{TimeZone, UUID}
```
Comment (ttim) [inline summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala]: nit: I think it should be with spaces (I think it's where your IDEA style settings going against our scalafmt setup).

**Review by ttim — APPROVED** [review]

**Review by johnynek — APPROVED** [review]

File: summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala (hunk 2 above)
Comment (piyushnarang) [reply summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala]: yeah seems like the Twitter defaults are set to not include the space :-).

**Comment by johnynek** [comment]
looks like develop already has this. closing.

**Pull request closed without merging**